finagle-redis/src/main/scala/com/twitter/finagle/redis/SentinelClient.scala

```diff
@@ -0,0 +1,85 @@
+package com.twitter.finagle.redis
+
+import com.twitter.finagle.builder.ClientBuilder
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.{ Service, ServiceFactory }
+import com.twitter.util.Future
+import org.jboss.netty.buffer.{ ChannelBuffers, ChannelBuffer }
+
+object SentinelClient {
+
+  /**
+   * Construct a sentinel client from a single host.
+   * @param host a String of host:port combination.
+   */
+  def apply(host: String): SentinelClient = SentinelClient(
+    ClientBuilder()
+      .hosts(host)
+      .hostConnectionLimit(1)
+      .codec(Redis())
+      .daemon(true)
+      .build())
+
+  /**
+   * Construct a sentinel client from a single Service.
+   */
+  def apply(raw: Service[Command, Reply]): SentinelClient =
+    new SentinelClient(raw)
+  
+  object Node {
+    def apply(props: Map[String, String]) = {
```

Comment:
scaladoc? in particular, props must have a role-reported key, or this will throw.